app/services/idv/profile_step.rb

```diff
@@ -0,0 +1,44 @@
+module Idv
+  class ProfileStep < Step
+    def complete?
+      idv_session.resolution.try(:success?) ? true : false
+    end
+
+    def complete
+      return if attempts_exceeded?
+      idv_session.params.merge!(params)
+      validate(params)
+      confirm if form_result[:success]
+      track_event
+      complete?
+    end
+
+    def attempts_exceeded?
+      attempter.exceeded?
+    end
+
+    private
+
+    def attempter
+      @_idv_attempter ||= Idv::Attempter.new(idv_form.user)
+    end
+
+    def confirm
+      idv_session.applicant = idv_session.applicant_from_params
+      idv_session.vendor = idv_agent.vendor
+      idv_session.resolution = idv_agent.start(idv_session.applicant)
+      attempter.increment
```

Comment:
Yes that is expected. The vendor tracks attempt velocity via the initial `start` of the IdV session. A user might make repeated attempts to verify financial information, e.g., but those attempts to not count as new attempts.

We may need to put additional throttling in per-step later but that is being worked out with the vendor.